server/db/models/transactions.js

```diff
@@ -0,0 +1,26 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Transactions = db.define('transactions', {
+  buyer: {
+    type: Sequelize.STRING,
```

Comment:
Buyer and Tutor should be associations to User, I think, not just string. Eg, Transactions.hasOne(User as "buyer") & Transactions.hasOne(User as "tutor")